code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1043,7 +1042,7 @@
 	return
 
 
-/datum/species/proc/spec_attack_hand(mob/living/carbon/human/M, mob/living/carbon/human/H, 	var/datum/martial_art/attacker_style = M.martial_art)
+/datum/species/proc/spec_attack_hand(mob/living/carbon/human/M, mob/living/carbon/human/H, datum/martial_art/attacker_style = M.martial_art)
```

Comment:
This argument seems pretty weird to me but apparently it works.
`datum/martial_art/attacker_style = M.martial_art`